test/arrays/sparse/copyRemoteSparse-cs.chpl

```diff
@@ -43,12 +43,11 @@ on Locales[numLocales-1] {
   var B = A2;
   stopCommDiagnostics();
   if countComms then printCommDiagnosticsTable();
+
   local {
-    for i in 1..9 {
-      for j in 1..9 {
-        if B.domain.contains(i,j) then
-          B[i,j] = 2;
-      }
+    for ij in D {
```

Comment:
And here, `D` can be accessed within `local` ? Maybe so thanks to RVF?

Reply:
Huh…  I didn't think of that, but did test it, but it doesn't seem like it should work, does it?  Thanks for the question and let me take another look.

Reply:
It seems that it's due to the `--remote-serialization` option.  At least, when I compile with `--no-remote-serialization` I get a locality violation on that line (well, actually, on the previous instance of it on line 22).

Such a powerful compiler!  :D